c2corg_api/views/user.py

```diff
@@ -198,6 +192,36 @@ def post(self):
             return None
 
 
+@resource(path='/users/request_password_change', cors_policy=cors_policy)
+class UserRequestChangePasswordRest(object):
+    def __init__(self, request):
+        self.request = request
+
+    @json_view(validators=[partial(validate_required_json_string, "email")])
+    def post(self):
+        request = self.request
+        email = request.validated['email']
+        user = DBSession.query(User).filter(User.email == email).first()
+        user.update_validation_nonce(VALIDATION_EXPIRE_DAYS)
+
+        try:
+            DBSession.flush()
+        except:
+            log.warning('Error persisting user', exc_info=True)
+            raise HTTPInternalServerError('Error persisting user')
+
+        # FIXME use attribute stored in the user object
+        lang = 'fr'
+
+        email_service = get_email_service(request)
+        nonce = user.validation_nonce
+        settings = request.registry.settings
+        link = settings['mail.request_password_change_url_template'] % nonce
+        email_service.send_request_change_password(lang, user, link)
```

Comment:
When sending the mail fails, can the use simply request the password change a 2nd time?